AISKU/Tests/testSnippet.ts

```diff
@@ -0,0 +1,332 @@
+
+
+/**
+ * DO NOT FIX BUGS WITH THE OBJECT RETURNED BY THIS HELPER
+ * -------------------------------------------------------
+ * YOU *SHOULD* CREATE A NEW VERSION FOR EACH SNIPPET VERSION UNLESS CHANGES ARE MINIMAL
+ * -------------------------------------------------------
+ * This is a helper that returns an object that is the same as the v5 snippet, if there is
+ * an issue with the tests because of the object this returns there is probably a bug during 
+ * initialization.
+ */
+export function createSnippetV5(snipConfig) {
```

Comment:
This returns the full version 5 snippet -- without adding to the window or attempting to drop a script tag

Reply:
It's basically a copy of the snippet with some parts commented out